clique/src/lib.rs

```diff
@@ -119,6 +119,13 @@ pub trait Dialer<A: Data>: Clone + Send + 'static {
     async fn connect(&mut self, address: A) -> Result<Self::Connection, Self::Error>;
 }
 
+pub trait NetworkCliqueMetrics {
+    fn set_present_incoming_connections(&self, present: u64);
+    fn set_missing_incoming_connections(&self, missing: u64);
+    fn set_present_outgoing_connections(&self, present: u64);
+    fn set_missing_outgoing_connections(&self, missing: u64);
```

Comment:
consistent with status and metrics in finality-aleph
```suggestion
    fn set_incoming_connections(&self, present: u64);
    fn set_missing_incoming_connections(&self, missing: u64);
    fn set_outgoing_connections(&self, present: u64);
    fn set_missing_outgoing_connections(&self, missing: u64);
```

Reply:
Done, but I'm already missing that equal alignment